src/gwasstudio/main.py

```diff
@@ -21,21 +21,20 @@
 @click.version_option(version=__version__)
 @cloup.option("-q", "--quiet", default=False, is_flag=True, help="Set log verbosity")
 @cloup.option_group(
-    "Dask options",
-    cloup.option(
-        "--distribute",
-        default=False,
-        is_flag=True,
-        help="Distribute the load to a Dask cluster",
-    ),
-    cloup.option("--local", is_flag=True, default=True, help="If the cluster needs to be kept local"),
-    cloup.option("--local_workers", help="Number of workers for local cluster", default=1),
-    cloup.option("--local_threads", help="Threads per worker for local cluster", default=4),
-    cloup.option("--local_memory", help="Memory per worker for local cluster", default="12GB"),
-    cloup.option("--minimum_workers", help="Minimum amount of running workers", default=10),
-    cloup.option("--maximum_workers", help="Maximum amount of running workers", default=100),
-    cloup.option("--memory_workers", help="Memory amount per worker", default="12GB"),
+    "Dask options HPC or Gateway",
+    cloup.option("--dask-distribute", default=False, is_flag=True, help="Distribute the load to a Dask cluster on a HPC that use SLURM"),
+    cloup.option("--minimum_workers", default=10, help="Minimum amount of running workers"),
+    cloup.option("--maximum_workers", default=100, help="Maximum amount of running workers"),
+    cloup.option("--memory_workers",  default="12GB", help="Memory amount per worker"),
     cloup.option("--cpu_workers", help="CPU numbers per worker", default=6),
+    cloup.option("--address", help="address in case a gateway is available", default=1),
```

Comment:
why default=1? Is it a string, no?

Reply:
Yeah, actually there is no default here so I put None